todo/src/routes/routes.ts

```diff
@@ -0,0 +1,133 @@
+import { server } from "../server"
+import { ResponseToolkit, Request } from "@hapi/hapi"
+import { tasks } from "../models";
+
+export function getRoute() {
+
+    // GET all Tasks
+    server.route({
+        method: 'GET',
+        path: '/task',
+        handler: (request: Request, h: ResponseToolkit) => {
+            return tasks;
+        }
+    });
+
+    // Only grab the todo that has the completed status of true
+    server.route({
+        method: 'GET',
+        path: '/task/completed',
+        handler: (request: Request, h: ResponseToolkit) => {
+            const found = tasks.some(task => true === task.complete);
+            if(found) {
+                return(tasks.filter(task => true === task.complete))
+            } else {
+                return (`No todo is completed`);
+            }
+        }
+    });
+
+    // Only grab the tasks that have a completed status of false
+    server.route({
+        method: 'GET',
+        path: '/task/incomplete',
+        handler: (request: Request, h: ResponseToolkit) => {
+            const found = tasks.some(task => false === task.complete);
+            if(found) {
+                return(tasks.filter(task => false === task.complete))
```

Comment:
Whats up with the parens?

`()`

Reply:
Request and h are both not being used and should not be in the parens, fixed that and fixed the others that had unused parameters.